### ISSUE TITLE
 Bump stable tag to 28.0.6 or even to 29.0.3

### ISSUE THREAD

**Issue by Zottel92** [nextcloud/docker#2252]
Hello,

since 25th Jun the rollout of Version 28.0.6 is the 100% rolled out version of V28 (with this commit: [update-server-commit](https://github.com/nextcloud-releases/updater_server/commit/c84443501a261c58db08793dc33b0c2432cfac63)

But even earlier this commit [update-server-commit](https://github.com/nextcloud-releases/updater_server/commit/a5014ed71702c4b1ff1e15e400ee4b01b16d8bd7) from the 23th of may the rollout from 28 to 29 was set to 100%.

Would it make sense to bump the stable tag to 28.0.6 or even to the right now 100% rolled out version of 29.0.3?
Or is there a bug blocking this?

Sorry that I always ask, but there seems to be no such overview available.
Otherwise I could look into and of possible help to work on them.
Or would it make sense to open Pull-Requests for bumping the stable version myself and then discuss possible blocking issues in the Pull-Request?

Thank you for the work on this container-image.
Best regards

**Comment by te-online** [comment]
Yes, I think it's an oversight that stable is not updated to `28.0.7` yet, because it has always worked that way. Not sure who can fix this? @J0WI?

**Comment by te-online** [comment]
In regards to bumping to `29.0.3`, I'm not sure if there are any guidelines on when a new version is stable enough for the `stable` tag, but I think there's plenty of life left in `28` (if we're looking at [the Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule), so I would expect an update of `stable` later this year.

**Comment by Zottel92** [comment]
Thanks for bumping the tag to 28.07.

I there an issue/a discussion that blocks 29?
As written above 29 seems to be rolled out 100% already on the update server.

**Comment by Zottel92** [comment]
I think I understand something wrong here.
Would bumping from 28.x.x to 29.y.y  skip a major version?
